temporalcli/devserver/log.go

```diff
@@ -25,6 +26,10 @@ func (s slogLogger) DPanic(msg string, tags ...tag.Tag) { s.Log(slog.LevelError,
 func (s slogLogger) Panic(msg string, tags ...tag.Tag)  { s.Log(slog.LevelError, msg, tags) }
 func (s slogLogger) Fatal(msg string, tags ...tag.Tag)  { s.Log(slog.LevelError, msg, tags) }
 
+func (s slogLogger) SLog() *slog.Logger {
+	return s.log
+}
```

Comment:
Forgive the probably-dumb Go question, but I don't understand how this is being used? Is there some magic somewhere that's calling this function?

Reply:
IIUC, it's being used in the server code. So we pass a `log.Logger` to the server, and server-side they see if it's also a `log.SLogWrapper` they call this to extract the slog.

Reply:
https://github.com/temporalio/temporal/blob/4a71548e03219f52203b059369d66286d30e85f5/common/log/slog.go#L55